src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -81,12 +82,37 @@ function StripePaymentForm({
 
     setIsLoading(true);
 
-    const { error } = await stripe.confirmPayment({
+    const stripePaymentMethodHandler = async (result) => {
+      if (result.error) {
+        // Show error in payment form
+        if (result.error.type === 'card_error' || result.error.type === 'validation_error') {
+          setMessage(result.error.message);
+        } else {
+          setMessage('An unexpected error occurred.');
+        }
+        setIsLoading(false);
+      } else {
+        // Otherwise send paymentIntent.id to your server
+        // TODO: refactor to fetch in service.js
+        const postData = JSON.stringify({ payment_intent_id: result.paymentIntent.id });
+        await getAuthenticatedHttpClient()
+          .post(`${process.env.STRIPE_RESPONSE_URL}`, { postData })
+          .catch(error => {
+            setMessage(error.message);
```

Comment:
Right now the message is shown to the user for debugging purposes, but we don't want to show this IRL, since the message can be something like -
`Axios Error (Response): 400 http://localhost:18130/payment/stripe/checkout {\"field_errors\":{\"first_name\":\"This ...`

Reply:
Is there a `DEBUG` flag or something you can test against to make sure this never shows up in production?  Otherwise committing something like this is a bit risky, we might forget it is there by the time we need to deploy.

Reply:
I will console.log this instead